spec/requests/internal/users_spec.rb

```diff
@@ -22,7 +22,7 @@
     end
   end
 
-  describe "GET internal/users/:id/edit" do
+  describe "GET /internal/users/:id/edit" do
```

Comment:
Not really related to this PR, but when I looked at this file I noticed some missing slashes, so I added them.